### DISCUSSION TITLE
Some question about tessellation with periodic boundary condition

### DISCUSSION THREAD

**Discussion by diuky** [neperfepx/neper#638] [Q&A] [answered]
Hi,

I'm a newly user for NEPER. I'm now trying to create the polycrystal tessellation using periodic boundary condition. I want to get the information of the neighboring seed. However, I found there are some indexes which is greater than the total number of seeds in the  information of the neighboring seed. I'm wondering how to find the corresponding seeds for these neighboring indexes. Thanks a lot!

**Comment by rquey** [comment, marked as answer]
The numbers of the neighboring *seeds* can indeed exceed the total number of grains (because some seeds are duplicated to compute the periodic tessellation).

If you are interested in the neighboring *grains*, use `ncells` instead of `nseeds` (e.g. in `-statcell`).
